editquality/utilities/extract_damaging.py

```diff
@@ -0,0 +1,263 @@
+r"""
+Gathers the reverted status for a set of revisions and
+prints a TSV to stdout of the format:
+
+<rev_id>\t<reverted>\t<reason>
+
+Usage:
+    extract_damaging -h | --help
+    extract_damaging <dump-file>... [--host=<url>]
+                                    [--start=<date>]
+                                    [--end=<date>]
+                                    [--trusted-groups=<groups>]
+                                    [--trusted-edits=<num>]
+                                    [--revert-radius=<revs>]
+                                    [--revert-window=<hrs>]
+                                    [--reverted-only]
+                                    [--check-blocked]
+                                    [--verbose]
+                                    [--rev-reverteds=<path>]
+
+Options:
+    -h --help                   Prints out this documentation.
+    <dump-file>                 Path to dump file.
+    --host=<url>                The host URL of the MediaWiki install where an
+                                API can be found.
+    --start=<timestamp>         Start time.
+    --end=<timestamp>           End time.
+    --reverted-only             Only mark reverted edits as potentially
+                                damaging
+    --revert-radius=<revs>      The maximum amount of revisions that a
+                                reverting edit can revert [default: 15]
+    --revert-window=<secs>      The maximum amount of time to wait for a
```

Comment:
We should probably let the user provide this in hours and do the conversion in code.